db/controllers/data.controller.js

```diff
@@ -0,0 +1,11 @@
+
+const pool = require('../connection.js')
+const queries = require('../queries/queries.js')
+
+exports.topicsRoute = (req, res) => {
+  pool.query(queries.getTopics, (err, results) => {
```

Comment:
We've been teaching the MVC structure so we'd like you to follow it here. Please extract this into a model and use your model to make queries to the database.